docs/TerminologyOperations.md

```diff
@@ -0,0 +1,134 @@
+`https://{LocalHost}/CodeSystem/$lookup?system={system}&code={code}`
+
+You can also use a `POST` request to look up a code by providing a parameters resource that contains a coding in the body of the request:
+`https://{LocalHost}/CodeSystem/$lookup`
+
+## Data Clean up
+
+Some of the FHIR resources downloaded from the FHIR specifications and US-Core Profile may require some clean up. It is suggested to clean the div element of the FHIR resources.
+
+Regex can be helpful here: 
+
+find     - "div": `<div (.*)>(.*)</div>`
+
+replace  - "div": `<div>PlaceHolder</div>`
+
+## Bundling US-Core PRofiles
+'''
+
+        using Hl7.Fhir.Model;
+
+        using Hl7.Fhir.Serialization;
+
+        using System.Runtime.CompilerServices;
+
+
+        var parser = new FhirJsonParser();
+
+        var bundle = new Bundle();
+        bundle.Type = Bundle.BundleType.Batch;
+
+        foreach (var file in Directory.GetFiles(@"Location of US-Core Profiles", "*.json"))
+        {
+            var fileName = Path.GetFileName(file);
+            var resourceType = fileName.Split('-')[0];
+            Resource resource = null;
+            string url = null;
+            switch (resourceType)
+            {
+                case "SearchParameter":
+                    resource = parser.Parse<SearchParameter>(File.ReadAllText(file));
+                    url = ((SearchParameter)resource).Url;
+                    break;
+                case "StructureDefinition":
+                    resource = parser.Parse<StructureDefinition>(File.ReadAllText(file));
+                    url = ((StructureDefinition)resource).Url;
+                    break;
+                case "ValueSet":
+                    resource = parser.Parse<ValueSet>(File.ReadAllText(file));
+                    url = ((ValueSet)resource).Url;
+                    break;
+                default:
+                    break;
+            }
+
+            if (resource != null)
+            {
+                bundle.Entry.Add(Utils.CreateBundleEntry(resource, url));
+            }
+        }
+
+        var serializer = new FhirJsonSerializer();
+
+        File.WriteAllText(@"outputFile.json", serializer.SerializeToString(bundle));
+
+        return 0;
+
+        public class Utils
+        {
+            public static Bundle.EntryComponent CreateBundleEntry(Resource resource, string url)
+            {
+                Bundle.EntryComponent entry = new Bundle.EntryComponent();
+                entry.Request = new Bundle.RequestComponent() { Method = Bundle.HTTPVerb.POST, Url = resource.TypeName };
+                entry.FullUrl = url;
+                entry.Resource = resource;
+                return entry;
+            }
+        }
+'''
+## Extra Information
+
+Intern Presentation on Terminology Service: https://microsoft-my.sharepoint.com/:p:/p/t-sadonchev/EdS_tgPIZV1Am0m7vog1zkABUdRqqLEybUhSy2MASVuARQ?e=PoRBey
```

Comment:
This won't be accessible externally, good link, but don't think it makes sense in this context